### DISCUSSION TITLE
Use sponsorship money to sponsor other projects

### DISCUSSION THREAD

**Discussion by JC5** [community/community#12217] [Other Feature Feedback, Questions, & Ideas]
I would like the ability to use the money I am sponsored with to sponsor other projects. It feels pointless to pay double transaction fees, transfer fees, processing fees, etc etc to sponsor somebody's project, if I could simply hand over some of my sponsor money to somebody else, before the money is even paid out to me.

It would save a lot of trees, that's for sure.

**Comment by jlord** [comment]
Hi! This is a great idea and something we're actively looking at. Glad to get this feedback, helps up prioritize :D

**Comment by sheredom** [comment]
I'd love this too - that any sponsors of my projects will pay **my sponsors** before giving me any money!

**Comment by gregmolnar** [comment]
Is there any update about this @jlord . It would be really useful for many who doesn't do open-source for the money but still get some sponsorship and they could pay it forward.

**Comment by Dnepr88888** [comment]
Доброго дня.я новачок але маю бажання та дуже сильну мотивацію.якщо в когось є можливість та бажання допомогти на цьому етапі як ₴₴₴хоч на телефон норм.!?а від наставника вообще не відказався б! всім зарання подякував! пишіть alexdnepr787@gmail.com 
Help me please 
[Telegram_Ecosystem.pdf](https://github.com/user-attachments/files/16392825/Telegram_Ecosy
stem.pdf)
Даже не могу з цим розібратися

**Comment by EmilStenstrom** [comment]
@jlord I would be interested in this as well for the django-components project. I have a monthly supporter, but now someone else is doing all the work, so I'd like to redirect the funds to them. Contacting the original funder is risky, and means I lose control of the ability to redirect to someone else in the future. Happy to give more details if you want.
